_config.yml

```diff
@@ -8,6 +8,12 @@ prbot_url: "https://canada-pr-bot.herokuapp.com/"
 wet_cdts_hosturl: "https://www.canada.ca/etc/designs/canada/cdts/gcweb"
 wet_cdts_version: "v4_0_30"
 
+active_emergency: true
```

Comment:
I think this could change from "emergency" to "campaign" and keep it general. That would make it easier to to reuse